README.md

```diff
@@ -95,6 +101,57 @@ If you have multiple Config Tokens as secrets, you can use all of them in the wo
 
+## Connect using OpenID Connect
+The sensitive connection details, such as the access token used by JFrog CLI on the JFrog platform, can be automatically generated by the action instead of storing it as a secret in GitHub. 
+This is made possible by leveraging the OpenID-Connect (OIDC) protocol. This protocol can authenticate the workflow issuer and supply a valid access token, requiring only the JF_URL environment variable.
+To utilize the OIDC protocol, there are several **one-time** steps that need to be performed:
+1. **[Configure an OIDC Integration](https://jfrog.com/help/r/jfrog-platform-administration-documentation/configure-an-oidc-integration)**: This phase sets an integration between the Action to the JFrog platform.
+
+| NOTE:                                                                                                                                                                                                             |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| The value specified as the 'Provider Name' should be used as the provider_name input in [step 4](#step4) below.                                                                                                   |
+| The 'Audience' field does not represent the 'aud' claim for insertion into the identity-mapping in [step 2](#step2) below. Only the claims included in the Claims Json created during step 2 will be validated.   |
+
+
+<div id="step2"/>
+
+2. **[Configure an identity mapping](https://jfrog.com/help/r/jfrog-platform-administration-documentation/configure-identity-mappings)**: This phase generates a reference token for authenticating against the JFrog platform. It involves defining the necessary details to enable server authentication of the action issuer and granting the issuer an appropriate access token. 
+   You have the flexibility to define any valid list of claims required for request authentication. You can check a list of the possible claims [here](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#understanding-the-oidc-token).
+   Example Claims JSON:
+   ```yml
+   {
+      "sub": "repo:my-user-name/project1:ref:refs/heads/main", 
+      "aud": "https://github.com/my-user-name",
+      "ref": "refs/heads/main",
+      "repository": "my-user-name/project1",
+      "iss": "https://token.actions.githubusercontent.com"
+   }
+   ```
+
+3. **Set required permissions**: In the course of the protocol's execution, it's imperative to acquire a JSON Web Token (JWT) from GitHub's OIDC provider. To request this token, it's essential to configure the specified permission in the workflow file:
```

Comment:
Let's separate between steps 1+2 and 3-5 to emphasize that 1+2 are server configurations and 3-5 are workflow configurations.